recipes/fastspar/meta.yaml

```diff
@@ -41,17 +44,10 @@ about:
   home: https://github.com/scwatts/fastspar
   license: GPLv3
   license_file: LICENSE
-  summary: Rapid and scalable correlation estimation for compositional data
-  description: |
-    ``FastSpar`` is a C++ implementation of the SparCC algorithm
-    which is up to several thousand times faster than the original
-    Python2 release and uses much less memory. The ``FastSpar``
-    implementation provides threading support and a *p*-value
-    estimator which accounts for the possibility of repetitious data
-    permutations.
+  summary: "Rapid and scalable correlation estimation for compositional data"
```

Comment:
Why did you delete the description?

Reply:
I don't know why I deleted this. I'll put it back right now.